release/ray_release/config.py

```diff
@@ -89,6 +96,25 @@ def validate_test(test: Test, schema: Optional[Dict] = None) -> Optional[str]:
         return str(e)
 
 
+def validate_test_cluster_compute(test: Test) -> Optional[str]:
+    from ray_release.template import load_test_cluster_compute
+
+    cluster_compute = load_test_cluster_compute(test)
+    return validate_cluster_compute(cluster_compute)
+
+
+def validate_cluster_compute(cluster_compute: Dict[str, Any]) -> Optional[str]:
```

Comment:
Nit. Anyscale support both Ebs configuration for global (.aws) as well as per node type (.head_node.aws I think....)

Reply:
Thanks, support added!